chart/templates/workers/worker-horizontalpodautoscaler.yaml

```diff
@@ -0,0 +1,34 @@
+################################
+## Airflow Worker HorizontalPodAutoscaler
```

Comment:
Should remove this now we have Keda as this never worked that well.

Reply:
done